Dockerfiles/manifests/generate.sh

```diff
@@ -131,7 +131,7 @@ for values in "$TMPDIR"/values-*.yaml; do
     rm -rf "${type:?}"
     mkdir "${type:?}"
 
-    helm template --kube-version 1.21 --namespace default datadog "${HELM_DATADOG_CHART:-datadog/datadog}" --values "$values" --output-dir "$TMPDIR/generated_$type"
+    helm template --kube-version 1.21 --namespace default datadog "${HELM_DATADOG_CHART:-datadog/datadog}" --set datadog.apiKey=DUMMY_API_KEY --values "$values" --output-dir "$TMPDIR/generated_$type"
```

Comment:
last request can be move to kube-version 1.24 ?

Reply:
Done :+1: